charts/zesty/values.yaml

```diff
@@ -1,208 +1,903 @@
+# @schema.root
+# additionalProperties: true
+# @schema.root
+# @schema
+# type: object
+# required: false
+# additionalProperties: true
```

Comment:
Why not set additionalProperties to false?
Are there global properties not specified in this file?